lib/rules/function-no-unknown/index.cjs

```diff
@@ -45,7 +48,10 @@ const rule = (primary, secondaryOptions) => {
 			return;
 		}
 
-		const functionsList = JSON.parse(fs.readFileSync(functionsListPath.toString(), 'utf8'));
+		// Read the file lazily for performance.
+		functionsListMemo ??= JSON.parse(fs.readFileSync(functionsListPath.toString(), 'utf8'));
```

Comment:
```suggestion
functionsListMemo ??= readJSONFile(functionsListPath.toString());
```

That's in `testUtils` folder though.

Reply:
Unfortunately, `testUtils` is unpublished:
https://github.com/stylelint/stylelint/blob/a6b9b92e3f7d09d4b044b2eb24b9007ab33230f1/package.json#L57